homeassistant/components/xiaomi_miio/select.py

```diff
@@ -160,6 +163,17 @@ class AttributeEnumMapping(NamedTuple):
         options=["forward", "left", "right"],
         entity_category=EntityCategory.CONFIG,
     ),
+    XiaomiMiioSelectDescription(
+        key=ATTR_MODE,
+        attr_name=ATTR_MODE,
+        name="Mode",
+        set_method="set_mode",
+        set_method_error_message="Setting the mode of the fan",
+        icon="mdi:fan",
+        translation_key="mode",
+        options=["silent", "auto", "favorite"],
```

Comment:
Should these also be uppercased? Or rather, stay lowercase and the translated strings to be uppercased?